modules/clickhouse/build.gradle

```
@@ -4,7 +4,15 @@ dependencies {
     api project(':testcontainers')
     api project(':jdbc')
 
+    compileOnly project(':r2dbc')
+    compileOnly 'com.clickhouse:clickhouse-r2dbc:0.6.0'
+
     testImplementation project(':jdbc-test')
-    testRuntimeOnly 'ru.yandex.clickhouse:clickhouse-jdbc:0.3.2'
+    testRuntimeOnly(group: 'com.clickhouse', name: 'clickhouse-jdbc', version: '0.6.0', classifier: 'http')
     testImplementation 'org.assertj:assertj-core:3.25.1'
+    testImplementation testFixtures(project(':r2dbc'))
+    testImplementation 'com.clickhouse:clickhouse-r2dbc:0.6.0'
```

according to the [docs](https://clickhouse.com/docs/en/integrations/java#r2dbc-driver) `http` can be chosen for a small jar, similar to clickhouse-jdbc, why was not applied here?

```suggestion
    testRuntimeOnly 'com.clickhouse:clickhouse-r2dbc:0.6.0'
```